### PR TITLE
docs: update branding, concepts

### PR DESCRIPTION
**Pull request by desimone**
## Summary

Changed some language. 

## Checklist

- [x] updated docs
- [x] add appropriate tag (`improvement` / `bug` / etc)
- [x] ready for review

### CODE REVIEW

File: docs/enterprise/about.md [hunk 1]
```diff
@@ -6,29 +6,29 @@ description: What does the Pomerium Enterpise Console offer?
...
-The Pomerium Enterprise Console lets you view traffic and logs, define routes and policies, and organize your service access all from an intuitive web interface.
+Pomerium Enterprise lets you view traffic and logs, define routes and policies, and organize your service access from an intuitive web interface.
 
-![Overview animation of the Pomerium Enterprise Console](./img/console-overview.gif)
+![Overview animation of Pomerium Enterprise](./img/console-overview.gif)
 
 ### Programmatic API
 
-Integrate Pomerium into your workflows by managing configuration from the programming language or infrastructure management tool of your choice.
+Integrate Pomerium into your workflows by managing configuration from the programming language or infrastructure management tool of your choice.Everything that is manageable in the Management GUI is also able to driven programmatically an API.
```
Comment (alexfornuto) [inline docs/enterprise/about.md:23]:
```suggestion
Integrate Pomerium into your workflows by managing configuration from the programming language or infrastructure management tool of your choice. Everything that is manageable in the Management GUI can also be driven programmatically through the API.
```

File: docs/enterprise/concepts.md [hunk 2]
```diff
@@ -1,16 +1,16 @@
...
 
 # Concepts
 
 ## Namespaces
 
-In the Pomerium Enterprise Console, a **Namespace** is a cornerstone organization unit. They are container objects that behave similar to a unix directory structure.
+In Pomerium Enterprise, a **Namespace** is a cornerstone organization unit. They are container objects that behave similar to a unix directory structure.
 
-In each Namespace, administrators can create organizational units where users and groups can be added. Namespaces enable fine-grained role based access control and management. The structure and hierarchy of namespaces empower teams to self-service the routes and policies pertinent to them. Namespaces can can also be used to optionally or mandatorily inherit from their parent permission or policies.
+In each Namespace, administrators can create organizational units where users and groups can be added. Namespaces enable fine-grained role based access control and management (RBAC) to managing Pomerium. The structure and hierarchy of namespaces empower teams to self-service the routes and policies pertinent to them. Namespaces can can also be used to optionally or mandatorily inherit from their parent permission or policies.
```
Comment (alexfornuto) [inline docs/enterprise/concepts.md:13]:
```suggestion
In each Namespace, administrators can create organizational units where users and groups can be added. Namespaces enable fine-grained role based access control and management (**RBAC**) to managing Pomerium. The structure and hierarchy of namespaces empower teams to self-service the routes and policies pertinent to them. Namespaces can can also be used to optionally or mandatorily inherit from their parent permission or policies.
```

**Review by alexfornuto — CHANGES_REQUESTED** [review]
I left a couple suggestions. In addition to those corrections, I want to confirm that we actually don't want to reference the "Enterprise Console", as such. See the thread in #docs for context.

File: docs/enterprise/concepts.md (hunk 2 above)
Comment (alexfornuto) [reply docs/enterprise/concepts.md:13]: This is part of the style I'd like to eventually formalize in a style guide: The fist time you introduce an acronym in a doc, you put the acronym in bold, within parens.

File: docs/enterprise/about.md (hunk 1 above)
Comment (alexfornuto) [reply docs/enterprise/about.md:23]: This is my attempt to fix the grammar, but if it changes the content too much, maybe we should 1:1 on what's being said here.

**Review by alexfornuto — APPROVED** [review]

**Review by travisgroth — APPROVED** [review]

**Pull request merged**